src/App.css

```diff
@@ -0,0 +1,51 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #22201c;
```

Comment:
try to use color variable

Reply:
This was pre defined css when created the setup for mockbee, I'll remove it in the next PR